server/server.go

```diff
@@ -1602,3 +1604,39 @@ func (s *Server) IsTTLConfigExist(key string) bool {
 	}
 	return false
 }
+
+// MarkRecovering mark pd that we're recovering
+// tikv will get this state during BR EBS restore.
+func (s *Server) MarkRecovering() error {
+	log.Info("mark recovering")
```

Comment:
I suggest printing the log after the operation is successful or printing the log in both the start and end place.

Reply:
those better be done using a middleware, won't do it here